src/osdc/Objecter.cc

```diff
@@ -119,6 +119,9 @@ enum {
   l_osdc_osdop_cmpxattr,
   l_osdc_osdop_rmxattr,
   l_osdc_osdop_resetxattrs,
+  l_osdc_osdop_tmap_up,
+  l_osdc_osdop_tmap_put,
```

Comment:
TMAP_PUT and TMAP_GET have been removed from use so they will never provide any stats. RBD still uses TMAP_UP, though, for v1 format images.

Reply:
Thank you very much for your revieded.I had removed the perf counter abort TMAP_PUT and TMAP_GET.